crates/bevy_ui/src/render/mod.rs

```diff
@@ -403,67 +310,28 @@ pub fn extract_uinode_images(
                 flip_x: image.flip_x,
                 flip_y: image.flip_y,
                 camera_entity,
-                border,
-                border_radius,
+                border: BorderRect::ZERO,
+                border_radius: ResolvedBorderRadius::ZERO,
                 node_type: NodeType::Rect,
```

Comment:
Intentional? Add a comment explaining

Reply:
Ah yeah I mixed this up, it's incorrect.